server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/sumcoin/sources/sumcoinindex/SumcoinindexResponse.java

```diff
@@ -0,0 +1,70 @@
+/*************************************************************************************
+ * Copyright (C) 2014-2016 GENERAL BYTES s.r.o. All rights reserved.
+ *
+ * This software may be distributed and modified under the terms of the GNU
+ * General Public License version 2 (GPL2) as published by the Free Software
+ * Foundation and appearing in the file GPL2.TXT included in the packaging of
+ * this file. Please note that GPL2 Section 2[b] requires that all works based
+ * on this software must also be made publicly available under the terms of
+ * the GPL2 ("Copyleft").
+ *
+ * Contact information
+ * -------------------
+ *
+ * GENERAL BYTES s.r.o.
+ * Web      :  http://www.generalbytes.com
+ *
+ ************************************************************************************/
+package com.generalbytes.batm.server.extensions.extra.sumcoin.sources.sumcoinindex;
+
+import com.generalbytes.batm.server.extensions.extra.sumcoin.sources.sumcoinindex.*;
+import java.math.BigDecimal;
+
+public class SumcoinindexResponse {
+    
+    private BigDecimal exch_rate;
+    private BigDecimal exch_rate_sell;
+    private BigDecimal exch_rate_buy;
+    private int error;
+    String error_msg;
```

Comment:
`private` modifier missing?

Reply:
Yup, added in https://github.com/GENERALBYTESCOM/batm_public/pull/137/commits/4cdca4845ba2aef8dc91834f92968e1594d39613